x-pack/solutions/security/plugins/security_solution/public/detection_engine/rule_creation/components/step_rule_actions/notification_action.tsx

```diff
@@ -49,7 +47,13 @@ export const FrequencyDescription: React.FC<{ frequency?: RuleActionFrequency }>
     return null;
   }
 
-  const { unit, value } = getTimeTypeValue(frequency.throttle);
+  const duration = parseTimeDuration(frequency.throttle);
+
+  if (!duration) {
+    return <DescriptionLine>{i18n.PERIODICALLY}</DescriptionLine>;
+  }
+
+  const { unit, value } = duration;
```

Comment:
How is the PR related to rule actions? If it's a different bugfix, let's open a separate PR for it.

Reply:
While it's not directly related to Rule Schedule it's related to the introduced time duration utilities. `getTimeTypeValue` was used in rule schedule functionality and it was superseded by time duration utilities. It was the only trivial place using `getTimeTypeValue`. Experience shows `getTimeTypeValue` will stay here for a while if it's not handled consistently with introduction of time duration utilities.

Reply:
rolled back